apps/docs/docs/quickstart.mdx

```diff
@@ -3,13 +3,29 @@ import TabItem from '@theme/TabItem';
 
 # Quickstart
 
+Setup is easy. 
+
+Once you understand how to use `ts-rest`, developer experience is like butter.  🧈
+
 ## Installation
 
-Install the core package
+1. Install the core package
 
 ```bash
-yarn add @ts-rest/core
+yarn add @ts-rest/core 
+--
+npm i @ts-rest/core
+```
+
+2. Enable `strictNullChecks` in your `tsconfig.json`:
```

Comment:
I think it would be better if we put this under the "zod" tab in the section below this, since this is not required if we are not using zod-based contracts.

Also, I believe the general recommendation is to use `strict` mode as a whole not just `strictNullChecks`, and we can link to the [README Requirements](https://github.com/colinhacks/zod#requirements) rather than that issue. We can include and extra comment insuring that `strictNullChecks` is not disabled as I believe this most applies to NestJS project's using the NestJS project generator.